confgenerator/testdata/valid/linux/logging-receiver_zookeeper_apache/input.yaml

```diff
@@ -0,0 +1,30 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+logging:
```

Comment:
Should the folder name be `confgenerator/testdata/valid/linux/logging-receiver_zookeeper` instead of `confgenerator/testdata/valid/linux/logging-receiver_zookeeper_apache`?

Reply:
Yeah I had played around with different sources at first. Forgot I renamed the folder.